syntax = "proto3";

package face_aggregation;

import "common.proto";

service FaceAnalysisService {
  rpc ReceiveImage(ImageToFaceServiceRequest) returns (DoneFlagToImageInputServiceResponse);
}